libs/feature-flags/src/lib/types.ts

```diff
@@ -0,0 +1,16 @@
+export interface User {
+  uuid: string
```

Comment:
Why uuid? Not sure if we have user uuids.

Reply:
I was under the impression we had some kind of GUID for every logged-in user?

But this could just a well be session id for unauthorized users. The name here is merely to emphasize that this should be a unique id.